CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.2.1 (2023-04-19)
+
+* `MaxMind::GeoIP2::Client` sets additional info in it's http client
```

Comment:
"its"

Also, what kind of info? Version information I'm assuming. We could probably say that!